### ISSUE TITLE
Meetup at 16

### ISSUE THREAD

**Issue by sohilpandya** [dwyl/learn-apple-watch-development#33]
Had a chat with Ines and agreed that we should have a meetup that takes place in 16 for Apple related things. 

Here is the event that I am proposing takes place on **Monday 8th May** that I'll post on the Founders & Coders Meetup Channel as a trial 

**Event:**
Learn iOS Development

**Description:**
Have you ever wanted to start learning development for iOS? We are hosting our very first meetup dedicated to iOS.

This is an open session, where you can pair up with others who are also starting out on the journey of learning how to build an app for the apple ecosystem. We'll provide you with a list of some very basic and open source resources that will help you get started on your journey.

Are you already an iOS developer? We'd love for you to join us and help people in the space who are new to the world of iOS. _Please feel free to contact us if you'd like to give a talk or intro session on anything related to iOS, we can certainly make space for that too!_

Can you please review this and let me know if anything needs changing. If you are happy with it then I can post it 🎉

**Comment by iteles** [comment]
You could maybe put out a call for mentors on the codebar slack channel? And find a few other ways to reach people via twitter etc.

Is clashing with Coding for Everyone a good idea though? Maybe take a look at the meetup  landscape that week and see if a Wednesday might be better (and people from the F&C community might benefit more as well)

**Comment by sohilpandya** [comment]
Yeah, I'll put it up for a Wednesday that week and see what kind of response we get, if it's overwhelming then we can invite some more mentors etc from codebar.

It's great to have those extra marketing channels! :)

**Comment by sohilpandya** [comment]
Here it is https://www.meetup.com/founderscoders/events/239541832/ 🎉 

@markwilliamfirth please have a look at it and let me know if I've spelled Focus Hub correctly 😉 
Don't want this to happen https://github.com/dwyl/hq/issues/249 😆

**Comment by sohilpandya** [comment]
**update**
we've got 29 people signed up for the meetup without any particular marketing efforts. C4E usually has a 30/40% turnout when compared to the people who've signed up. 

Do you reckon it's worth putting this on the codebar slack channels and then also the FAC gitter channels?

**Comment by iteles** [comment]
@sohilpandya :tada: Assuming most of those people are beginners, it would be great to get some mentors!

**Comment by sohilpandya** [comment]
@iteles I am not part of the codebar slack channel, would you know the best way to get onto that channel? 😬

**Comment by markwilliamfirth** [comment]
@sohilpandya I'm a member - if you paste in a comment what I should say and tag me I can post it

**Comment by sohilpandya** [comment]
Thanks for that @markwilliamfirth 

Hey Codebar,

We need your help! We're starting a new meetup dedicated to [learning iOS](https://www.meetup.com/founderscoders/events/239541832/) we've had a tremendous response from the general public as there clearly seems to be a need for an event where people can come together to pair up on learning iOS but unfortunately, we don't have many mentors who would be able to help around in the space if people were looking for advice or just got stuck on something they are working on. 

Hence we'd like to put a message out on this channel to all iOS mentors to come and join us if you can on Wednesday 10th May from 18:30 - 20:00 at [Focus Hub](https://www.google.co.uk/maps/place/16+Palmers+Rd,+London+E2+0SY/data=!4m2!3m1!1s0x48761d2695efb1b1:0x30455a5f7ce8535d?sa=X&ved=0ahUKEwjgz76zxNnTAhVMLcAKHenXCZ0Q8gEIKTAB).

PS. If you'd like to come along to pair up and learn iOS please come along too! 🎉
